### ISSUE TITLE
Is this extension no longer maintained?

### ISSUE THREAD

**Issue by dmusican** [fwcd/vscode-kotlin#165]
Currently, this extension has a significant bug that prevents it from working for Windows users (#158) for recent versions of VSCode (1.92+).

The VSCode team posted an issue about this on July 10 (#153), a bug report was filed (#158); and two independent pull requests have been posted that fix the bug. This is great --- but there has been no response at all from the maintainer, so the extension remains broken. Is this project no longer being maintained?

I'm teaching a class that makes use of this extension, and while we're ok for now because all of my students have already installed it, we can't install it on any new Windows machines. We're trying to evaluate whether we can depend on this extension going forward for the future, or whether we should switch to a different solution. @fwcd, have you moved on from this project to doing other things?

**Comment by mpolovincak-cen55773** [comment]
I have the same question. It is super useful extension, using it every day, waiting for merging pull requests and release them (as they contain features I need). Using on macOS, so I can still use it.

**Comment by fwcd** [comment]
Yeah, sorry, I'm a bit short on time these days, but try to keep everything running at least. Sorry for the delay on this issue.

**Comment by willnode** [comment]
Thank u!
